app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapViewClient.java

```diff
@@ -0,0 +1,14 @@
+package com.teamagam.gimelgimel.app.view.viewer.cesium;
+
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+/**
+ * A {@link WebViewClient} to control our {@link CesiumMapView}
+ */
+public class CesiumMapViewClient extends WebViewClient {
```

Comment:
Id suggest that you rename this class to express that it disables links.
Something like DisableLinkWebViewClient or OverrideAnyUrlWebViewClient